core/src/main/java/com/linecorp/armeria/internal/common/NoopKeepAliveHandler.java

```diff
@@ -64,4 +64,10 @@ public boolean needsDisconnection() {
 
     @Override
     public void increaseNumRequests() {}
+
+    @Override
+    public void tryNotifyConnectionActive() {}
```

Comment:
What do you think of just adding a `decreaseNumRequests` at the location suggested by @ikhoon at this comment?
https://github.com/line/armeria/pull/5580#discussion_r1642858108